docs/setup/secure-settings.asciidoc

```diff
@@ -7,6 +7,9 @@ keystore, and the `kibana-keystore` tool to manage the settings in the keystore.
 
 NOTE: All commands here should be run as the user which will run Kibana.
 
+
+NOTE: Only some settings are designed to be read from the keystore. However, the keystore has no validation to block unsupported settings when being added. Adding unsupported settings to the keystore causes Kibana to fail to start. To see whether a setting is supported in the keystore, look for a "Secure" qualifier the setting reference. Furthermore, setting values will be validated upon startup, in case you're getting validation errors, review your configuration values.
```

Comment:
This feels too verbose for a Note section.
I want to express 3 things: 
 - Settings are not validated when persisted
 - Settings are validated when Kibana is started up
 - Extraneous settings, or non-JSON-parseable settings might cause errors on start up